pkgs/applications/misc/albert/default.nix

```
@@ -73,7 +73,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     homepage = "https://albertlauncher.github.io";
     changelog = "https://github.com/albertlauncher/albert/blob/${finalAttrs.src.rev}/CHANGELOG.md";
-    license = licenses.gpl3Plus;
+    license = licenses.unfree;
```

Adding context may be useful here:

```suggestion
    # See: https://github.com/NixOS/nixpkgs/issues/279226
    license = licenses.unfree;
```